### ISSUE TITLE
Poor performance with MNIST

### ISSUE THREAD

**Issue by PramitSaha** [ESHyperscale/HyperscaleES#1]
I was running the MNIST example and it gives me 53% accuracy. MNIST sgd gives 93%. I am wondering if I did something wrong! Cannot find any reference for this.

**Comment by bsarkar321** [comment]
Thanks for pointing this out! This was an old test script with suboptimal hyper-parameters, which we have removed from this repo to avoid future confusion.

For reference, we are able to get to SGD-level performance by increasing "group size" by many orders of magnitude and keeping naive hyperparameters (with a linear decay of learning rate and/or sigma)

**Comment by Daulbaev** [comment]
Could you please upload an updated version of the MNIST training? I still can't find the right set of hyperparameters to obtain a good quality.
